### ISSUE TITLE
About head_fc in BCLModel_32.

### ISSUE THREAD

**Issue by Jacky-H-uang** [PanHaulin/ConCutMix#1]
Hi, your research is so excellent, but when i run the code i face some confused problmes in the BCL_Model_32. About the `self.head_fc = nn.Sequential(nn.Linear(num_classes, dim_in), nn.BatchNorm1d(dim_in), nn.ReLU(inplace=True),  nn.Linear(dim_in, feat_dim))` the first Linear it should look like `nn.Linear(dim_in, dim_in)`or something else？
